pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -610,40 +594,43 @@ func copyPGCertsAndMakeURL(
 	t test.Test,
 	c cluster.Cluster,
```

Comment:
nit: could you add a docstring to this function? also, coud you move the `c.ExternalPGURL()` into this function? It's currently used to create the last input arg, but it could be done within the function.